lib/Checks/FunctionCommaSpaceAfterCheck.vala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2016 elementary LLC. (https://github.com/elementary/Vala-Lint)
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program; if not, write to the
+ * Free Software Foundation, Inc., 59 Temple Place - Suite 330,
+ * Boston, MA 02111-1307, USA.
+ *
```

Comment:
Please remove this line, if you don't want to name you as the author. :-) Just for consistency with the the copyright-header-cleanup a few minutes ago 😄